.github/workflows/ui_test.yml

```
@@ -0,0 +1,64 @@
+name: Testing workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Postgres code
+        uses: ikalnytskyi/action-setup-postgres@v5
+
+      - name: Postgres setup
+        env:
+          PGHOST: localhost
+          PGDATABASE: evershop
+          PGUSER: postgres
+          PGPASSWORD: postgres
+        run: |
+          createdb --owner $PGUSER $PGDATABASE
+          psql -c "ALTER USER $PGUSER WITH PASSWORD '$PGPASSWORD'"
+
+      - name: Checkout code
+        uses: actions/checkout@v2
```

```suggestion
      - name: Checkout code
        uses: actions/checkout@v2
```
May be this step at first position
```suggestion
      - name: Checkout code
        uses: actions/checkout@v2
```
```suggestion
      - name: Checkout code
        uses: actions/checkout@v2
```

It would be good to have this step at first

---

okay